scipy/cluster/tests/test_vq.py

```diff
@@ -413,9 +413,9 @@ def test_kmeans_and_kmeans2_random_seed(self, xp):
             # test for kmeans
             res1, _ = kmeans(data, 2, seed=seed1)
             res2, _ = kmeans(data, 2, seed=seed2)
-            xp_assert_close(res1, res2, xp=xp)  # should be same results
+            xp_assert_close(res1, res2)  # should be same results
```

Comment:
Do we ever want to pass `xp` into `xp_assert_close`? It generally won't be an `array_api_compat` version of the namespace, and that causes trouble in `xp_assert_close`.